File: 1.Fundamental-Level/3.OOP/00.OOP-Exercises/04.InheritanceAndAbstraction-1/GoldenEditionBook.cs
using System;

public class GoldenEditionBook : Book
{
    private const decimal Rate = 1.3m;

    public GoldenEditionBook(string title, string author, decimal price)
        : base(title, author, price)
    {
    }

    public override decimal Price
    {
        get
        {
            return base.Price * Rate;
        }
    }
}

File: Java-Development/Java-OOP-Basics/Java-OOP-Basics-Solutions/src/inheritance/_02_BookShop/GoldenEditionBook.java
package inheritance._02_BookShop;

public class GoldenEditionBook extends Book {

    public GoldenEditionBook(String title, String author, Double price) {
        super(title, author, price);
    }

    @Override
    public Double getPrice() {
        return super.getPrice() + super.getPrice() * 0.3;
    }
}
